### ISSUE TITLE
Home - jury members - adding a button with the link 

### ISSUE THREAD

**Issue by klaudia2209** [ecommerceberlin/ecommercegermanyawards.com#133]
Homepage - there are shown jury members, but not all off them. 

Is it possible to add a button which will redirect the user to 'JURY MEMBERS PAGE' with all the jurors? 

I am not sure about the text on the button, it could be 'SEE ALL JURORS' or 'SEE MORE' maybe?

The reason is that on homepage there are only 8 jury members visible and it looks like there is no more of them. 

<img width="1437" alt="image" src="https://user-images.githubusercontent.com/118257251/203536807-b920dfb0-b06d-4d84-8717-f4b566029fcd.png">

**Comment by az** [comment]
@klaudia2209 yes because the goal of the homepage is not showing jurors but gathering submissions ... the full list is of course available via https://ecommercegermanyawards.com/jurors

**Comment by az** [comment]
@klaudia2209 im prefixing the title with (IDEA) ... do you remember the labels we discussed ? :)

**Comment by az** [comment]
@klaudia2209 i will check in a few days if i can add such button

**Comment by klaudia2209** [comment]
Sure, it's a submission, but if someone stops only on homepage, probably won't go to 'jury members'. I think that it could be a good idea, because even jurors will look for themselves on the website. 

Yeah, sorry, I forgot. 

(MAINTENANCE) - when should I use this one? 
(IDEA) - clear 
(EMAIL) - is it dedicated to system e-mails? 
(BUG) - I understand
